ShearTests/InMemoryHistoryStorageTests.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  InMemoryHistoryStorageTests.swift
+//  ShearTests
+//
+//  Created by Andreas Kompanez on 23.02.20.
+//  Copyright © 2020 Andreas Kompanez. All rights reserved.
+//
+
+import XCTest
+
+@testable import Shear
+
+class InMemoryHistoryStorageTests: XCTestCase {
+    var storage: InMemoryHistoryStorage!
+
+    override func setUp() {
+        super.setUp()
+        
+        storage = InMemoryHistoryStorage()
+    }
+
+    override func tearDown() {
+        storage = nil
+        
+        super.tearDown()
+    }
+    
+    func testDuplicateDetection() {
+        let item0 = HistoryItem(kind: .plainText("0"))
+        let item1 = HistoryItem(kind: .plainText("0"))
+        XCTAssertEqual(item0, item1)
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/895e50bbb85826ea77037e4a1c1e5468d960c4fc/ShearTests/InMemoryHistoryStorageTests.swift#L32"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>InMemoryHistoryStorageTests.swift:32</code></del></td>
    </tr>
  </tbody>
</table>